modules/tinymce/src/plugins/table/test/ts/browser/TableInsertionTest.ts

```diff
@@ -0,0 +1,39 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+
+import * as TableTestUtils from '../module/test/TableTestUtils';
+
+describe('browser.tinymce.plugins.table.TableCellPropsStyleTest', () => {
```

Comment:
The file name here is incorrect. Maybe `TableRemoveTrailingBrTest.ts` might be a clearer file name?

Reply:
For this test, I am considering naming the file in a generic way. This will allow for future tests involving insertion to be added here easily.